app/services/analytics_events.rb

```diff
@@ -4953,4 +4953,17 @@ def webauthn_update_name_submitted(
       **extra,
     )
   end
+
+  # A user has been moved deactivated due to not visting the post office before the deadline
```

Comment:
Let's add to to. your sentence; moved `to` deactivated
Also, do you think it would be helpful to clarify that this user was flagged on the ThreatMetrix check?

Reply:
Yeah thats a good callout @gina-yamada this method only fires if they have been flagged for fraud review.